modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Ratings/Rating.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using JetBrains.Annotations;
+using Volo.Abp;
+using Volo.Abp.Auditing;
+using Volo.Abp.Domain.Entities;
+
+namespace Volo.CmsKit.Ratings
+{
+    public class Rating : BasicAggregateRoot<Guid>, IHasCreationTime, IMustHaveCreator
+    {
+        public virtual Guid? TenantId { get; protected set; }
+        
+        public virtual string EntityType { get; protected set; }
+
+        public virtual string EntityId { get; protected set; }
+        
+        public virtual short StarCount { get; protected set; }
+        
+        public virtual Guid CreatorId { get; set; }
+
+        public virtual DateTime CreationTime { get; set; }
+
+        protected Rating()
+        {
+            
+        }
+        
+        public Rating(
+            Guid id,
+            [NotNull] string entityType,
+            [NotNull] string entityId,
+            short starCount, 
+            Guid creatorId,
+            Guid? tenantId = null
+        )
+            : base(id)
+        {
+            EntityType = Check.NotNullOrWhiteSpace(entityType, nameof(entityType), RatingConsts.MaxEntityTypeLength);
+            EntityId = Check.NotNullOrWhiteSpace(entityId, nameof(entityId), RatingConsts.MaxEntityIdLength);
+            SetStarCount(starCount);
+            CreatorId = creatorId;
+            TenantId = tenantId;
+        }
+
+        public virtual void SetStarCount(short starCount)
+        {
+            if(starCount <= 5 && starCount > 0)

```

Comment:
Instead of such a magical value, extract a constant (in a `RatingConsts` class). If the component we are using is supporting more stars, we can create a `public static int MaxRating { get; set; }` instead of a constant. If it supports minimum stars, we can also make it changeable like that.